expat/lib/libexpat.def

```diff
@@ -72,7 +72,7 @@ EXPORTS
   XML_ResumeParser @64
   XML_GetParsingStatus @65
 ; added with version 2.1.1
-; XML_GetAttributeInfo @66
+  XML_GetAttributeInfo @66
```

Comment:
Please revert the uncommenting here — the default should remain at disabled.

Reply:
Addressed review comments in latest patch set please do review.